modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/deployer/ObjectDefinitionDeployerImpl.java

```diff
@@ -109,27 +109,34 @@ public synchronized List<ServiceRegistration<?>> deploy(
 			return Collections.emptyList();
 		}
 
+		_objectDefinitionInstanceKey =
+			objectDefinition.getRESTContextPath() +
+				objectDefinition.getCompanyId();
+
 		ObjectScopeProvider objectScopeProvider =
 			_objectScopeProviderRegistry.getObjectScopeProvider(
 				objectDefinition.getScope());
 
 		Map<Long, ObjectDefinition> objectDefinitions =
-			_objectDefinitionsMap.get(objectDefinition.getRESTContextPath());
+			_objectDefinitionsMap.get(_objectDefinitionInstanceKey);
 
 		if (objectDefinitions == null) {
 			objectDefinitions = new HashMap<>();
 
 			_objectDefinitionsMap.put(
-				objectDefinition.getRESTContextPath(), objectDefinitions);
+				_objectDefinitionInstanceKey, objectDefinitions);
 
 			_excludeScopedMethods(objectDefinition, objectScopeProvider);
 
 			_componentInstancesMap.put(
-				objectDefinition.getRESTContextPath(),
+				_objectDefinitionInstanceKey,
 				Arrays.asList(
 					_objectEntryApplicationComponentFactory.newInstance(
 						HashMapDictionaryBuilder.<String, Object>put(
 							"liferay.jackson", false
+						).put(
+							"liferay.object.definition.company.id",
+							objectDefinition.getObjectDefinitionId()
 						).put(
 							"liferay.object.definition.id",
 							objectDefinition.getObjectDefinitionId()
```

Comment:
Why is this the object definition id? shouldn't it be the company id?

Reply:
Oh! I forgot to delete that property. I added it making some tests but it is no longer necessary. Thanks!!